app/models/operations/load_service_areas.rb

```diff
@@ -115,14 +121,14 @@ def parse_text(input)
     end
 
     def parse_boolean(value)
-      return true   if value == true   || value =~ /(true|t|yes|y|1)$/i
-      return false  if value == false  || value =~ /(false|f|no|n|0)$/i
+      return true   if value == true   || value =~ /(true|t|yes|Yes|y|1)$/i
+      return false  if value == false  || value =~ /(false|f|no|No|n|0)$/i
```

Comment:
we have case-insensitive match here already. why do we need capitalized `Yes and No` ?

Reply:
```
irb(main):056:0> 'Yes' =~ /(true|t|yes|y|1)$/i
=> 0
irb(main):057:0> 
irb(main):058:0> 'yes' =~ /(true|t|yes|y|1)$/i
=> 0
irb(main):059:0> 'Invalid' =~ /(true|t|yes|y|1)$/i
=> nil
irb(main):060:0>
```

I can see it's working. are you seeing anything different? @Rohithone

Reply:
when I process the file it didn't work, Let me recheck and change acccordingly